benchmarks/graph_infra/arkouda.graph

```diff
@@ -22,6 +22,24 @@ files: groupby.dat, groupby.dat, groupby.dat, groupby.dat
 graphtitle: Groupby Performance
 ylabel: Performance (GiB/S)
```

Comment:
Not from this PR, but just noticing this is `GiB/S` instead of `GiB/s` (uppercase S)